client/client_test.go

```diff
@@ -918,6 +930,77 @@ func testListTarget(t *testing.T, rootType string) {
 	assert.Equal(t, currentTarget, newCurrentTarget, "current target does not match")
 }
 
+func testListTargetWithDelegates(t *testing.T, rootType string) {
```

Comment:
It uses the type for the delegation key so it's probably worth doing it with both key types.